src/components/Header/components/PeopleSearch/index.js

```diff
@@ -334,17 +349,33 @@ export default class GordonPeopleSearch extends Component {
                       }),
                     )}
                   </Paper>
-                ) : isOpen && this.state.suggestions.length === 0 &&
-                  this.state.query.length >= MIN_QUERY_LENGTH ? (
-                  // Styling copied from how renderSuggestion is done with
-                  // only bottom padding changed and 'no-results' class used
-                    <Paper square className="people-search-dropdown">
-                      {this.renderNoResult()}
-                    </Paper>) : null}
-              </span>
-            )}
-          </Downshift>
-        );
+                ) : (
+                  <Paper square className="people-search-dropdown">
```

Comment:
How does it work when a ternary operator has 5 operands? In other words, in which cases do each of these colon-separated operands take effect?

Reply:
It is just like two nested `if` statements, it works like
```
if (condition1) {              // condition1 ? (
  if (condition2) {            //   condition2 ? (
    return (<FirstThing />);   //     <FirstThing />
  } else {                     //   ) : (
    return (<SecondThing />);  //     <SecondThing />
  }
} else {                       // ) : (
  return (<ThirdThing />);     //   <ThirdThing />
}                              // )
```
I tried it out a bunch in CodePen before putting in the real code, because I saw this same thing done in other parts of 360 and I was also confused about that at first

Reply:
You could easily nest the second condition inside the outer `else` case too, just by putting that part after the first colon. It is far more versatile than I thought at first

Reply:
Oh, okay! I did not notice the other `?`s